compiler/luci/pass/src/FuseInstanceNormPass.test.cpp

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
@seanshpark 
I've added this file for internal functions testing, is it ok?

I am also thinking about negative tests for this optimization (creating whole graph in luci IR).
Is it ok to create graph in this file, and not by recipe?

Reply:
> I've added this file for internal functions testing, is it ok?

Seems OK. I'm not sure what concern you have for adding tests :)

> Is it ok to create graph in this file, and not by recipe?

Yes, graph from codes (adding nodes with C++ codes) is also OK.